paint/util/paint_mappings.py

```diff
@@ -452,3 +452,11 @@
 SPLIT_KEY = "Split"
 DISTANCE_WINTER = "distance_winter"
 DISTANCE_SUMMER = "distance_summer"
+
+
+# Keys for Dataset file identifiers.
+CALIBRATION_RAW_IMAGE_IDENTIFIER = "_raw.png"
+CALIBRATION_CROPPED_IMAGE_IDENTIFIER = "_cropped.png"
+CALIBRATION_FLUX_IMAGE_IDENTIFIER = "_flux.png"
+CALIBRATION_FLUX_CENTERED_IMAGE_IDENTIFIER = "_flux_centered.png"
+CALIBRATION_PROPERTIES_IDENTIFIER = "-calibration-properties.json"
```

Comment:
This is inconsistent now. Calibration has dashes, while all the other have underscores. Should it remain this way or do we want to unify the names to dashes only (to avoid OS-specific problems)?